.github/workflows/main.yml

```diff
@@ -39,9 +39,9 @@ jobs:
         - { comp: gcc         , arch: aarch64 , opts: -Wno-psabi, status: 1 }
         - { comp: gcc         , arch: arm     , opts: -Wno-psabi, status: 1 }
         - { comp: gcc         , arch: ppc64   , opts: -Wno-psabi, status: 1 }
-        - { comp: gcc         , arch: sve128  , opts: -Wno-psabi, status: 0 }
-        - { comp: gcc         , arch: sve256  , opts: -Wno-psabi, status: 0 }
-        - { comp: gcc         , arch: sve512  , opts: -Wno-psabi, status: 0 }
+        - { comp: gcc         , arch: sve128  , opts: -Wno-psabi -DEVE_USE_PCH=OFF, status: 0 }
```

Comment:
I thought you wanted that in the toolchain files?

Reply:
or even remove that since we have g++12 no?

Reply:
Unfortunately, still crashes. Yeah, let me move to toolchain.

Reply:
OK